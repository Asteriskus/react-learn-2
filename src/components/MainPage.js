import React from 'react';
import ExpenseList from './ExpenseList';
import ExpensesFilter from './ExpensesFilter';

const MainPage = () => (
    <div>
        <ExpensesFilter />
        <ExpenseList />
    </div>
);

export default MainPage;