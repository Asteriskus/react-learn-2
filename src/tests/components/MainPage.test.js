import React from 'react';
import { shallow } from 'enzyme';
import MainPage from '../../components/MainPage';

test('Should render MainPage', () => {
        const wrapper = shallow(<MainPage/>);
        expect(wrapper).toMatchSnapshot();
    }
);