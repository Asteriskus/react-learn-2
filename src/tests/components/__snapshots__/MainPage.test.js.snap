// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render MainPage 1`] = `
<div>
  <Connect(ExpensesFilter) />
  <Connect(ExpenseList) />
</div>
`;
